recursion.js

```diff
@@ -27,7 +33,33 @@ const factorial = n => {
 
 console.log(factorial(5));
 
+
 // write the above function in a recursive way.
+const recursiveFactorial = n => { 
+  // let i = 2;
+  // if (n === 0) return 1;
+  // return n * recursiveFactorial(n-1);
 
```

Comment:
Hey, just in case you guys lost it, here's the top-to-bottom approach incrementing down from n:
```js
function recursiveFactorial(n) {
    if (n === 0) return 1;
    return n * recursiveFactorial(n - 1);
}
console.log(recursiveFactorial(5));
```